scripts/bootstrap_snp.sh

```
@@ -0,0 +1,187 @@
+    echo "Installation complete."
+    if [ "$NEW_KERNEL_VERSION" != "$CURRENT_KERNEL" ] && [ "$choice" = "2" ]; then
+        echo "NOTE: A system reboot is still required to activate all changes."
+    fi
+}
+
+source_common
+
+if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
+  echo "Script was sourced"
+else
+  bootstrap $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)